pkgs/applications/graphics/mandelbulber2/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, mkDerivation, fetchFromGitHub, libpng, gsl, libsndfile, lzo,
+qmake, qttools, qtbase, qtmultimedia,
+opencl ? true, opencl-clhpp ? null, ocl-icd ? null }:
+
+assert opencl -> opencl-clhpp != null;
+assert opencl -> ocl-icd != null;
+
+mkDerivation rec {
+  repo = "mandelbulber2";
+  name = "${repo}-${version}";
```

Comment:
You can drop this line once you set `pname` and `version` as `name = "${pname}-${version}";` is already implied.